src/camera.py

```diff
@@ -3,209 +3,182 @@
 
 Author: Steven Dropiewski
 """
-from os import path
-import sys
+from numpy import clip
 import time
 
-FILE_PATH = path.dirname(path.realpath(__file__))
-SUNFOUNDER_PATH = "SunFounder_PiCar-V/remote_control/remote_control/driver"
-sys.path.append(FILE_PATH + "/../" + SUNFOUNDER_PATH)
-
-from picar.SunFounder_PCA9685 import Servo
 from picar import filedb
+from picar.SunFounder_PCA9685 import Servo
 import picar
 
+
 class Camera:
     """
-    Camera class to control panning and tilting.
-
-    Creates camera object that can be operated on using set_pan and set_tilt.
-    set_pan operates [0,180] degrees. set_tilt operates [0,150] degrees. The 
-    current values for pan and tilt in degrees can be returned using the get_pan
-    and get_tilt methods. To change the offset you can may use calibrate_pan
-    and calibrate_tilt methods. The methods will automatically update the values
-    in the config file. All method parameters take absolute degrees.
-    
-    :param bus_number: The bus number the hat connects to.
+    Creates a camera object that can be panned and tilted.
+
+    The camera can pan in [0, 180] degrees, and can tilt in [0, 150] degrees.
+
+    :param bus_number: The bus number of the servos.
     :type bus_number: int
     """
 
     PAN_CHANNEL = 1
-    """Channel on PiCar HATS that connects to pan servo."""
+    """Pan servo's I2C channel"""
     TILT_CHANNEL = 2
-    """Channel on PiCar HATS that connects to tilt servo."""
+    """Tilt servo's I2C channel"""
     MAX_PAN_DEGREE = 180
-    """The maximum value to be written to pan servo (in degrees)."""
+    """The maximum value to be written to pan servo."""
     MAX_TILT_DEGREE = 150
-    """The maximum value to be written to tilt servo (in degrees)."""
+    """The maximum value to be written to tilt servo."""
     MIN_DEGREE = 0
-    """The minimum value to be written to both pan and tilt servo (in degrees)."""
-    MAX_CALI_DEGREE = 180
-    """The maximum value to calibrate pan and tilt (in degrees)."""
+    """The minimum value to be written to both pan and tilt servo."""
+    MAX_CALI_PAN_DEGREE = 180
+    """The maximum value to calibrate pan."""
+    MAX_CALI_TILT_DEGREE = 150
+    """The maximum value to calibrate tilt."""
     MIN_CALI_DEGREE = -180
-    """The minimum value to calibrate pan or tilt (in degrees."""
-    OFFSET_TO_SERVO = 0
-    """The offset value passed to the servo object."""
+    """The minimum value to calibrate pan or tilt."""
 
     def __init__(self, bus_number=1):
+        self.OFFSET_TO_SERVO = 0
```

Comment:
I think we can do the same for pan_channel and tilt_channel since they're also only ever used in init. Probably better if they're parameters with 1 and 2 (respectively) as the defaults.